src/UrdFactory.sol

```diff
@@ -2,6 +2,7 @@
 pragma solidity 0.8.21;
 
 import {UniversalRewardsDistributor} from "./UniversalRewardsDistributor.sol";
+import {IUniversalRewardsDistributor} from "./interfaces/IUniversalRewardsDistributor.sol";
```

Comment:
Then this is not used anymore ^^

Reply:
It's used in the return type but I can return a `UniversalRewardsDistributor` you're right